packages/manager/apps/dedicated/client/app/ip/components/list/list.controller.js

```diff
@@ -556,6 +568,12 @@ export default class IpListController {
       });
     };
 
+    $scope.goToVrack = function goToVrack(ipBlock) {
+      if (ipBlock.type === VRACK && ipBlock.routedTo?.serviceName) {
+        self.goToVrack(ipBlock.routedTo?.serviceName);
+      }
```

Comment:
it means `routeTo` can be null and in that case what's the behavior. Does the clock do nothing ?

Reply:
when routeTo is null normaly the type is not VRACK the button already disabled, just to be on the safe side when is null yes the clock do nothing